pkg/ceph/mon/config.go

```diff
@@ -84,7 +85,7 @@ func getMonKeyringPath(configDir, monName string) string {
 
 // get the path of a given monitor's data dir
 func getMonDataDirPath(configDir, monName string) string {
-	return filepath.Join(getMonRunDirPath(configDir, monName), fmt.Sprintf("mon.%s", monName))
+	return filepath.Join(getMonRunDirPath(configDir, monName), "data")
```

Comment:
I thought mkfs created a `store` subdir. can you verify? I want to avoid `mon0/data/store/`